pkg/gen/input/renovate/internal/file/renovate.json.template

```diff
@@ -0,0 +1,36 @@
+{{- $interval := .Interval -}}
+{{- $language := .Language -}}
+{{- $reviewer := .Reviewer -}}
+{
+  "extends": [
+    "config:base"
+  ],
+  {{- if eq $language "go" }}
+  "packageRules": [
+    {
+      "matchPackagePatterns": [".*giantswarm.*"],
+      "groupName": "giantswarm modules"
+    },
+    {
+      "matchPackagePatterns": ["^k8s.io", "^sigs.k8s.io"],
+      "groupName": "k8s modules"
+    },
+    {
+      "excludePackagePatterns": [".*giantswarm.*" ,"^k8s.io", "^sigs.k8s.io"],
+      "groupName": "etc modules"
+    },
+    {
+      "matchPackagePatterns": ["^k8s.io"],
+      "allowedVersions": "< 1.21.0"
+    },
+    {
+      "matchPackageNames": ["sigs.k8s.io/controller-runtime"],
+      "allowedVersions": "< 0.7.0"
```

Comment:
Is there a way to have additional rules but just for a specific repo?

It would be nice to have `helm.sh/helm/v3 < 3.6.0`. We only need that in helmclient but we might have similar requirements in other repos. WDYT?

Reply:
We can extend github meta data for that. What do you think to extend https://github.com/giantswarm/github/tree/master/repositories/override ? 
For example, we can merge `override/chart-operator/renovate.json` with the default `renovate.json`

Reply:
Nice, I'd forgotten about the overrides. I think that will work well.